guide/java/feeds.md

```diff
@@ -67,15 +69,15 @@ protected void disconnectSensors() {
 An SSH feed executes a command over ssh periodically. An example is shown below:
 
 {% highlight java %}
-private SshFeed feed;
+private AbstractCommandFeed feed;
 
 //@Override
```

Comment:
@drigodwin I uncommented it similar to `disconnectSensors` method mentioned in the example.